src/pages/Welcome/Welcome.scss

```
@@ -0,0 +1,82 @@
+@import "../../Styles/common.scss";
+
+.Welcome {
+    width: 100%;
+    height: 1048px;
+    padding: 100px 0;
+    h2 {
+        height: 190px;
+        background-color: #faf9f7;
+        font-size: 48px;
+        font-weight: 300;
+        letter-spacing: -0.04em;
+        @include flex-center;
+    }
+    .contentWrapper {
+        height: 533.800px;
+        margin: 44px 305px 0px;     
+        .engDescription {
+            //height: 40px;
+            font-size: 16px;
+            color: #666;
+            line-height: 40px;
+            font-weight: 400;
+            @include flex-center;
+        }
+        .korDescription {
+            //height: 50.4px;
+            font-size: 30px;
+            font-weight: 400;
+            display: block;
+            @include flex-center;
+        }
+        .joinBox {
+            //height: 392.4px;
+            border-top: solid 1px black;
+            margin-top: 50px;
+            @include flex-center; 
+            .joinTotal {
```

```suggestion
            @include flex-center; 
            
            .joinTotal {
```
새로운 셀렉터 앞에서 한줄 띄워주세요